### DISCUSSION TITLE
Help with Login / Register script setup

### DISCUSSION THREAD

**Discussion by vitalybrandes** [OrchardCMS/OrchardCore#7429] [Q&A]
If someone can assist me with **script**  Login / Register users from OIDC (using orchard core different tenant).
"**Site 1**" store user information (uname, email, roles etc)
i would like to store that information on "**Site 2**"  as well if possible sync roles and email periodically.

**Comment by vitalybrandes** [comment]
Any help? ☹️

**Comment by willnationsdev** [comment]
@vitalybrandes This isn't directly related to your question, but it is similar. I wanted to implement a Content Syndication module that allowed information entered in one tenant to be stored in another tenant and then arbitrarily shared to all tenants (though it related to ContentItems rather than Login/Registration) -> #7455 .

I would look at the OrchardCore source code for the Login controller/endpoint and see if there are any event handlers that execute after someone logs in (I'm sure there are). You should be able to implement that class and have it take the registered/logged-in user's data and migrate it somewhere else. Based on the comments in the linked issue, you can see how you're able to create a local scope with access to another tenant's service collection (to get its ISession or ILoginManager or whatever).
